libs/explorers/shared/src/lib/components/about/about.component.html

```diff
@@ -0,0 +1,5 @@
+<explorers-wiki-driven
+  title="About"
+  [wikiId]="wikiId"
+  [className]="className"
+></explorers-wiki-driven>
```

Comment:
AboutComponent and NewsComponent share the same structure and styling so they are now thin wrappers around WikiDrivenComponent.

Reply:
That's a good idea.  I was thinking of doing that initially but thought adding a wrapper might be cleaner.  Since I am already specifying the wikiId, I think you're right.  The component wrappers just seem like additional overhead and flexibility we don't need.  I'll just add heroTitle (instead of wikiTitle) and keep the main logic in WikiDrivenComponent.  That removes the need for any inputs and boilerplate.